### PR TITLE
Use reshape to re-implement squeeze

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@huningxin, @ibelem, PTAL, thanks!

File: nsnet2/nsnet2.js [hunk 1]
```diff
@@ -50,15 +50,21 @@ export class NSNet2 {
     });
     const [gru94, gru93] = this.builder_.gru(transpose31, weight192, recurrentWeight193, frames, this.hiddenSize,
         {bias: bias194, recurrentBias: recurrentBias194, initialHiddenState: initialState92, returnSequence: true});
-    const squeeze95 = this.builder_.squeeze(gru93, {axes: [1]});
+    // Use reshape to implement squeeze(gru93, {axes: [1]});
+    const squeezed95_shape = gru93.shape();
```
Comment (huningxin) [inline nsnet2/nsnet2.js:54]: Be consistent with the following one that is `squeeze158_shape`
```suggestion
    const squeeze95_shape = gru93.shape();
```

File: style_transfer/fast_style_transfer_net.js [hunk 2]
```diff
@@ -25,8 +25,14 @@ export class FastStyleTransferNet {
 
   buildInstanceNormalization_(conv2D, variableMul, variableAdd) {
     if ('instanceNormalization' in this.builder_) {
-      return this.builder_.instanceNormalization(conv2D,
-          {scale: this.builder_.squeeze(variableMul), bias: this.builder_.squeeze(variableAdd)});
+      // Use reshape to implement squeeze(variableMul); and squeeze(variableAdd);
+      const mul_shape = variableMul.shape();
```
Comment (huningxin) [inline style_transfer/fast_style_transfer_net.js:29]: Be compact?
```suggestion
      const mul_shape = variableMul.shape().filter(dim => dim !== 1);
```

File: style_transfer/fast_style_transfer_net.js [hunk 3]
```diff
@@ -25,8 +25,14 @@ export class FastStyleTransferNet {
...
+      const add_shape = variableAdd.shape();
```
Comment (huningxin) [inline style_transfer/fast_style_transfer_net.js:30]: ditto

File: nsnet2/nsnet2.js (hunk 1 above)
Comment (Honry) [reply nsnet2/nsnet2.js]: Thanks, fixed.

File: style_transfer/fast_style_transfer_net.js (hunk 2 above)
Comment (Honry) [reply style_transfer/fast_style_transfer_net.js]: It works, thanks!

File: style_transfer/fast_style_transfer_net.js (hunk 3 above)
Comment (Honry) [reply style_transfer/fast_style_transfer_net.js]: Done.

**Comment by Honry** [comment]
@huningxin, thanks for your comments, PTAL again.

**Review by huningxin — APPROVED** [review]
LGTM, thanks @Honry !

**Pull request merged**